pkg/action/show.go

```diff
@@ -113,7 +113,7 @@ func (s *Show) Run(chartpath string) (string, error) {
 			printer.Execute(&out, s.chart.Values)
 		} else {
 			for _, f := range s.chart.Raw {
-				if f.Name == chartutil.ValuesfileName {
+				if f.Name == chartutil.ValuesFilename {
```

Comment:
Why are there change of things like `ValuesfileName` to `ValuesFilename` and 'ChartfileName` to `ChartFilename`? These seem unrelated and get into the semantics of how we talk about things internally. Might I suggest reverting those changes.

Reply:
Because filename is one word 

ref.: https://en.wikipedia.org/wiki/Filename

It was done in a separate commit

Reply:
i guess can be either, ill revert 
![image](https://user-images.githubusercontent.com/24660299/206281335-9a6bfe58-20e7-4207-a53c-671a47d3ec28.png)